src/restic/node.go

```diff
@@ -378,7 +409,48 @@ func (node Node) sameContent(other Node) bool {
 			return false
 		}
 	}
+	return true
+}
 
+func (node Node) sameAcl(other Node) bool {
+	if node.Xattrs == nil {
```

Comment:
This is not a good test. When one node has the attribute set to nil, and the other to the empty slice (which is `!= nil`), it is considered differently. Just use the length test in line 422 below, `len(node.Attributes) != len(other.Attributes)`.